.claude/CLAUDE.md

```
@@ -15,11 +15,12 @@
 - `packages/core` → npm: `orbkit`
 - `packages/editor` → npm: `@orbkit/editor`
 
-## Build
-`bun build` for ESM + CJS + `.d.ts` outputs.
-
-## Test
-`bun test`
+## Commands
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)